index.html

```diff
@@ -1,312 +0,0 @@
-                                                structural system to the more complex multidegree-of-freedom and
-                                                continuous
-                                                systems,
-                                                including random
-                                                vibrations, nonlinear systems, and aeroelastic phenomena.
-                                          </p>
-                                          <div class="linkButton">
-                                                <button id="pic-link-button"><a href="blogPost.html"
-                                                            target="_blank">Learn more!</a></button>
-                                          </div>
-                                    </div>
-                                    <div class="articleBox">
-                                          <h3>Plane flies Upside-down!</h3>
-                                          <img class="articlePicture" id="articlePic12" src="images/002.jpg" width=40%
-                                                height=40%>
-
-                                          <p>In this way, the student can be guided from the simple concept of a
-                                                single-degree-of-freedom
-                                                structural system to the more complex multidegree-of-freedom and
-                                                continuous
-                                                systems,
-                                                including random
-                                                vibrations, nonlinear systems, and aeroelastic phenomena.
-                                          </p>
-                                          <div class="linkButton">
-                                                <button id="pic-link-button"><a href="blogPost.html"
-                                                            target="_blank">Learn more!</a></button>
-                                          </div>
-                                    </div>
-
-                              </div>
-                        </div>
-                  </div>
-            </div>
-
-            <div class="footerContainer">
-                  <div class="footer">
-
-
-                        <div class="footerTitle">QUESTIONS OR COMMENTS? <SPAN id="footerTitleBold">GET IN TOUCH:</SPAN>
-
-                        </div>
-
-                        <div class="formSection">
-
-                              <form action="action_page.php">
-                                    <div>
-                                          <input type="text" id="fname" name="firstname" placeholder="Your name.."
-                                                required>
-
-                                          <input type="text" id="email" name="lastname" placeholder="Your email"

```

Comment:
There is some controversy over whether or not to use the placeholder attribute. According to MDN, it is an [archived attribute](https://developer.mozilla.org/en-US/docs/Archive/Mozilla/XUL/Attribute/placeholder) that should not be used in [new websites](https://developer.mozilla.org/en-US/docs/Archive). Here are also a couple articles talking about this topic: [Here](https://www.smashingmagazine.com/2018/06/placeholder-attribute/) and [here](https://developer.paciellogroup.com/blog/2011/02/html5-accessibility-chops-the-placeholder-attribute/).